src/tools/crossgen2/ILCompiler.ReadyToRun/IL/ReadyToRunILProvider.cs

```diff
@@ -102,7 +102,13 @@ public override MethodIL GetMethodIL(MethodDesc method)
                         return result;
                 }
 
-                MethodIL methodIL = EcmaMethodIL.Create((EcmaMethod)method);
+                // PInvoke precompilation is limited to the system module for now
```

Comment:
When in largeversion bubble world (and the bubble includes CoreLib, you should also generate P/Invokes. An analysis of the methods jitted on startup is showing that there are a rather large chunk in System.IO.FileSystem that are hit on startup.

Reply:
Would the runtime look at them? There seemed to be deliberate blocking in PrepareILBasedCode.

Reply:
No, but we could allow it in the prototype branch...